src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -238,6 +238,11 @@ MavlinkReceiver::handle_message(mavlink_message_t *msg)
 		handle_message_obstacle_distance(msg);
 		break;
 
+	case MAVLINK_MSG_ID_TUNNEL:
+		handle_message_mavlink_tunnel(msg);
```

Comment:
Nitpick, please keep the naming convention consistent (handle_message_NAME).

handle_message_mavlink_tunnel -> handle_message_tunnel